_sources/GeneralIntro/Algorithms.rst

```diff
@@ -11,15 +11,70 @@
    :prefix: intro-2-
    :start: 1
 
+.. index:: programming language, algorithm, pseudocode,
+           program
+
 Algorithms
 ----------
+           
+If problem solving is a central part of computer science, then the solutions that you create through
+the problem solving process are also important.  In computer science, we refer to these solutions
+as **algorithms**.  An algorithm is a step by step list of instructions that if followed exactly will solve the problem under consideration.
+
+For example, an algorithm to sort a list of numbers in order from least to greatest might look like this:
```

Comment:
the algorithm provided is for calculating the area of a circle, not for sorting...

Reply:
I've fixed this. Sorry about that...